pkg/secretless/config/v2/config_http.go

```diff
@@ -2,24 +2,31 @@ package v2
 
 import (
 	"errors"
-
 	"github.com/go-ozzo/ozzo-validation"
 	"gopkg.in/yaml.v2"
 )
 
 type httpConfig struct {
-	AuthenticationStrategy   string   `yaml:"authenticationStrategy"`
 	AuthenticateURLsMatching []string `yaml:"authenticateURLsMatching"`
 }
 
 // HTTPAuthenticationStrategies are the different ways an http service
 // can authenticate.
-var HTTPAuthenticationStrategies = []string{
+var HTTPAuthenticationStrategies = []interface{}{
```

Comment:
This list should be dynamically generated from plugin list that has `http/` as prefix for now

Reply:
We'd have to append our internal ones to that list, correct?

Reply:
Yup. I'm going to defer that to another PR. There's a broader issue where we need to have validation happen in v2 instead of delegating it to v1. We also need to have this list of http service connectors but in a way that doesn't coupled the v1 way of doing things. A lot of this should be resolved as part of the plugin refactor.